src/jmcomic/jm_downloader.py

```
@@ -1,6 +1,29 @@
 from .jm_option import *
 
 
+def catch_exception(field_name):
+    def deco(func):
+        def wrapper(self, *args, **kwargs):
+            try:
+                return func(self, *args, **kwargs)
+            except Exception as e:
+                detail: JmBaseEntity = args[1]
```

Ruff (0.8.2)

`JmBaseEntity` may be undefined, or defined from star imports

(F405)